### ISSUE TITLE
No option for people outside the United States!

### ISSUE THREAD

**Issue by Artificial-Intelligence8** [natewong1313/bird-bot#16]
I was going to use this bot, however, I noticed that under the countries tab there's only one option and that's the US. Could you add options for countries such as Canada? Thanks!

**Comment by sonicer105** [comment]
I second wanting a Canada option

**Comment by SoulEst** [comment]
Software has bestbuy.com and walmart.com hardcoded in dozens of places. Some major rework will have to be done to allow order from BestBuy and Walmart in other countries.

**Comment by yulayula16** [comment]
> Software has bestbuy.com and walmart.com hardcoded in dozens of places. Some major rework will have to be done to allow order from BestBuy and Walmart in other countries.

Not necessarily, you can just make copies of those files and just be sure to change the URL and encoded strings for that particular site.

**Comment by yulayula16** [comment]
Software has bestbuy.com and walmart.com hardcoded in dozens of places. Some major rework will have to be done to allow order from BestBuy and Walmart in other countries.

Not necessarily, you can just make copies of those files and just be sure to change the URL and encoded strings for that particular site.
